itests/kit/sectors.go

```diff
@@ -0,0 +1,16 @@
+package kit
+
+import (
+	"github.com/filecoin-project/go-state-types/abi"
+
+	"github.com/filecoin-project/lotus/chain/actors/policy"
+)
+
+// EnableLargeSectors enables 512MiB sectors. This is useful in combination with
+// mock proofs, for testing larger transfers.
+func EnableLargeSectors() {
```

Comment:
We should probably use `t.Cleanup` here to return the global policy state back to the default.